src/identity.interfaces.ts

```diff
@@ -78,6 +99,10 @@ export interface IIdentityAPIIdentityChangeData {
 }
 
 export interface IIdentityRequest {
+    combineUserIdentities(
+        currentUserIdentities: UserIdentities,
+        newUserIdentities: UserIdentities
```

Comment:
I can't remember where we discussed this, but I remember us having a conversation where we thought `currentUserIdentities` and `newUserIdentities` was more appropriate and we should move towards that name.

I can go either way on this.

Reply:
I can't remember that discussion.  I'd say let's stick to the former for now since that's how the code currently is.  If there's a reason to change it in the future (maybe when refactoring identity.js --> identity.ts), then it'll come up again.